cmake/FreecivDependencies.cmake

```diff
@@ -170,3 +170,9 @@ endif()
 if (FREECIV_ENABLE_FCMP_CLI OR FREECIV_ENABLE_FCMP_QT)
   find_package(SQLite3 REQUIRED)
 endif()
+
+# Add Sphinx as a soft requirement so we can generate man pages and add to installer.
```

Comment:
Add a message when Sphinx is not found, similar to what we do for stack unwinding libraries?